src/main/pg/pg_ids.h

```diff
@@ -127,7 +127,7 @@
 #define PG_PINIOBOX_CONFIG 530
 #define PG_USB_CONFIG 531
 #define PG_SDIO_CONFIG 532
-#define PG_DISPLAY_PORT_CRSF_CONFIG 533
+#define PG_DISPLAY_PORT_CRSF_CONFIG 533  // no longer required
```

Comment:
If we leave this in, can we please mark this as 'never released', to avoid having to search through the history in the future?